bin/endtoend-build-tests

```diff
@@ -20,17 +20,18 @@ for t in $tests; do
   cp "${__root}/test/endtoend/$t/index.html" "${__root}/test/endtoend/$t/dist"
   browserify "${__root}/test/endtoend/$t/main.js" \
     -o "${__root}/test/endtoend/$t/dist/bundle.js" \
+    -t envify \
     -t babelify
 done
 
 # Speeecial tests that need custom builds.
 pushd "${__root}/test/endtoend/create-react-app"
   npm install
-  npm run build
+  REACT_APP_ON_TRAVIS="${TRAVIS:-}" npm run build
 popd
 pushd "${__root}/test/endtoend/typescript"
   mkdir -p dist
   cp "${__root}/packages/uppy/dist/uppy.min.css" dist/
   cp index.html dist/
-  browserify main.ts -p [ tsify --target ES3 ] -o dist/bundle.js
+  browserify main.ts -t envify -p [ tsify --target ES3 ] -o dist/bundle.js
```

Comment:
`envify` is needed for that `REACT_APP_ON_TRAVIS` above?

Reply:
More or less—this one in particular is to inline `CI` and `TRAVIS`, and create-react-app injects the `REACT_APP_` var by default.